MovieSharing/Thumbnail.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Thumbnail.swift
+//  MovieSharing
+//
+//  Created by Mariam Korashi on 01.10.21.
+//
+
+import Foundation
+
+struct Thumbnail: Decodable {
```

Comment:
- [ ] consider renaming to something like `ThumbnailContainer` as that makes it more clear that this contains data for different thumbnails